src/App.js

```diff
@@ -139,6 +143,27 @@ class App extends Component {
   };
 
   render() {
+
+    if ( 
+      this.props.availableOrganisations.length === 0 && 
+      this.props.isFetchingOrganisations === false &&
+      this.props.timesFetchedOrganisations > 0
+    ) {
+      const norolesMessage = this.props.intl.formatMessage({ id: "authorization.no_roles_message", defaultMessage: "Dear user, \nYou seem not to be in any organisations that can acces the management pages. \nTherefore you are redirected to the mainpage" });
+      alert(norolesMessage);
+      // should redirect to demo.lizard.net on prod
+      window.location = "/";
+    }
+
+    const currentHomeAppIcon = appIcons.find(icon => {
+      return window.location.href.includes(icon.linksTo.path)
+    });
+    if (currentHomeAppIcon && !doArraysHaveEqualElement(this.props.selectedOrganisation.roles, currentHomeAppIcon.requiredRoles)) {
+      const redirectMessage = this.props.intl.formatMessage({ id: "authorization.redirected_based_onrole", defaultMessage: "You do not have the rights to acces this data under the selected organisation. \nYou will be redirected" });
```

Comment:
'Access' instead of 'acces'. And isn't it better to select an organisation where the user does have the right rights? Anyway it shouldn't be possible to get into this situation.

Reply:
Text changed.
A user can come here by url. 
Or by chaning organisation (we can discuss if this should be possible)

Reply:
acces changed
- possible to get here by url
- or by changing organisation (should that be allowed?)